Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Description: This patch disables the tests building, avoiding an ftbfs (because
 the rating support is disabled in other patch).
--- a/discover/CMakeLists.txt
+++ b/discover/CMakeLists.txt
@@ -1,4 +1,4 @@
-add_subdirectory(tests)
+#add_subdirectory(tests)
 add_subdirectory(icons)
 
 include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_SOURCE_DIR}/libmuon ${CMAKE_CURRENT_BINARY_DIR}/..)
--- a/libmuon/CMakeLists.txt
+++ b/libmuon/CMakeLists.txt
@@ -22,7 +22,7 @@ MACRO(QT4_ADD_DBUS_INTERFACE _sources _i
 
 ENDMACRO(QT4_ADD_DBUS_INTERFACE)
 
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
 set(muon_LIB_SRCS
     Application.cpp
